docs/body.md

```diff
@@ -465,246 +268,310 @@ The following device level metrics are collected:
+- /OpenStack/nova-scheduler
+- /OpenStack/rabbitmq-server
 
-1. In Control Center, open the service page for your Zenoss instance.
-2. Click "Assign" for the node IP assignment (port 55672).
-3. Select the appropriate IP from the "IP" select box on the "Assign IP" modal dialog.
-4. Click "Assign IP".
-5. Repeat for the management console IP Assignment (45672).
 
-NOTE: Control Center services will not be able to restart successfully until the IP addresses are added.
+Zenpack Installation
+-----------------------
 
-### OpenStack Ceilometer Configuration Steps
+### Zenoss Configuration Steps - First-Time Installation
 
-Zenoss relies upon a Ceilometer dispatcher plugin to ship raw event and
-metering data from Ceilometer to Zenoss for storage in the Zenoss event
-and performance databases. This integration is done by publishing
-messages to Zenoss's RabbitMQ server.
+If you are installing the ZenPack for the first time, install as per
+usual ZenPack installation.
 
-This dispatcher should be installed on all nodes running any ceilometer,
-but particularly those running ceilometer-collector or
-ceilometer-agent-notification.
+### Zenoss Configuration Steps - ZenPack Upgrades
 
-Ceilometer_zenoss must be installed on all ceilometer nodes in the openstack
-environment.  To install the latest released version from RPM:
+If this is the first time you are upgrading to 3.0.0+ from a version of the
+zenpack 2.4.0 or earlier, there are no special steps required,
+nor any changes required on the OpenStack side.
 
-Download the appropriate RPM and install as usual: [https://github.com/zenoss/ceilometer_zenoss/releases]()
+If upgrading to 2.4.0, from a prior version, please consult the old
+documentation, *old_ceilometer.md*, located in the docs folder of this
+[git repo](https://github.com/zenoss/ZenPacks.zenoss.OpenStackInfrastructure/tree/master/docs).
 
-Alternatively, the module may be installed from source as follows:
 
-``` {.bash]
-$ sudo pip -q install --force-reinstall https://github.com/zenoss/ceilometer_zenoss/archive/master.zip 
-```
+### Post-ZenPack Installation
 
-It is then necessary to install a modified `/etc/ceilometer/event_definitions.yaml` file that
-is included in ceilometer_zenoss:
+*   Make sure to restart the Zproxy service to see the new UI elements
+    *User Domain Name* and *Project Domain Name* in the *Add OpenStack Endpoint*
+    Dialog.
 
-``` {.bash}
-$ sudo cp /usr/lib/*/site-packages/ceilometer_zenoss/event_definitions.yaml /etc/ceilometer/
-```
+*   Zenoss doesn't assign IP addresses for services proxy-zenopenstack and
+    RabbitMQ-Ceilometer, you must assign the correct collectors' host to those
+    services. Then you must restart those services in ControlCenter.
 
-Then, ensure that the configuration options output by the `openstack_amqp_config`
-script previously are added to /etc/ceilometer/ceilometer.conf file on all
-openstack nodes.
+    Once [CC-4195](https://jira.zenoss.com/browse/CC-4195) is completed,
```

Comment:
I would not mention CC-4195 or this possible future change.  CC-4195 has not even been looked at yet.  It's quite possible that this will be treated as an enhancement request or that it will be deemed to be invalid.  It's quite possible that this step will never change.